### PR TITLE
Trading

### PR DESCRIPTION
**Pull request by rohit-ganguly**
created an article where i pretend i know wtf is going on in this industry for a little bit

### CODE REVIEW

File: src/guides/career/trading.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+
```
Comment (varughese) [inline src/guides/career/trading.md:1]: delete this new line at the top

Comment (Zmwang622) [reply src/guides/career/trading.md:1]: why dont you just delete it

Comment (Zmwang622) [reply src/guides/career/trading.md]: @varughese I made the changes can you merge this

Comment (varughese) [reply src/guides/career/trading.md]: forgot i could edit inline

Comment (Zmwang622) [reply src/guides/career/trading.md]: lol yah can you approve or are there still issues

**Pull request merged**